vault/external_tests/raftha/raft_ha_test.go

```diff
@@ -53,6 +54,18 @@ func TestRaft_HA_NewCluster(t *testing.T) {
 	})
 }
 
+func Test_RaftHA_Recover_Cluster(t *testing.T) {
```

Comment:
This should have a GoDoc (comment beginning with the test name explaining what the test does above the test). It doesn't need to be long since you've already got a great description over at `testRaftHARecoverCluster`.